src/bindGlobalEventListeners.ts

```diff
@@ -45,16 +46,14 @@ export function onDocumentMouseMove(): void {
  * TODO: find a better technique to solve this problem
  */
 export function onWindowBlur(): void {
-  const {activeElement}: {activeElement: any} = document;
-  const instance = activeElement._tippy;
+  const {activeElement} = document;
 
-  if (
-    activeElement &&
-    activeElement.blur &&
-    instance &&
-    !instance.state.isVisible
-  ) {
-    activeElement.blur();
+  if (isReferenceElement(activeElement)) {
+    const instance = activeElement._tippy!;
+
+    if (activeElement instanceof HTMLElement && !instance.state.isVisible) {
```

Comment:
`instanceof` checks don't work in iframes

Reply:
`isType(activeElement 'HTMLElement')` util should work

Reply:
`isType` will not work, for example, for a button need to pass `'HTMLButtonElement'` as type, then `isType` returns `true`. Maybe revert `activeElement.blur` condition?

Reply:
> Maybe revert activeElement.blur condition

Seems safest option yes